assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -529,6 +529,7 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
     const filterVars = createCapiObjectFromStateVars(simLife.currentState, simLife.domain);
     if (filterVars && Object.keys(filterVars)?.length !== 0) {
       handleIFrameSpecificProperties(simLife.currentState, simLife.domain);
+      sendFormedResponse(simLife.handshake, {}, JanusCAPIRequestTypes.VALUE_CHANGE, filterVars);
```

Comment:
wait so now we need to send them all at once unaltered, then resend them each one at a time possibly parsing the string values?

Reply:
Oops. it was a mistake. I think I was investigating some other issue and checked-in this by mistake. reverted the changes